src/storage.js

```diff
@@ -22,11 +22,14 @@ export default class Store {
    static removeTask = (element) => {
      let tasks = Store.getToDos();
      tasks = tasks.filter((item) => item.index.toString() !== element.id.toString());
+     console.log(element.id);
      let i = 1;
      tasks.forEach((task) => {
        task.index = i;
        i += 1;
      });
+     window.location.reload();
```

Comment:
- [ ] Great work on implementing the changes requested by the previous reviewer but I guess your page is reloading after every deletion which is not really a professional approach to this :thinking: Your page should not reload at all on any operation like **deletion**, **insertion**, **updation**. To achieve this you can use your **displayTasks** method from your **displaytodo.js** file :+1:

**Question**
Now you can say that the previous reviewer talked about the reload page and he was testing my application by reloading the page.

**Answer**
He was just testing your data so that after reloading or opening your application on a new page will it show the same data. You were deleting your todos without reloading the page before so you can use somehow similar kind of technique to achieve this requirement :+1:

**Note: You can see this [ToDoList](https://hammas-saleem.github.io/ToDoList/) behavior by creating and removing todos from it :+1:**